src/DataMiner/SLManagedAutomation/UIBuilder.cs

```diff
@@ -280,6 +280,31 @@ public class UIBuilder
 		/// </example>
 		public UIBuilder AppendButton(string destVar, string displayText) { return null; }
 
+
+		/// <summary>
+		/// Adds a button to this dialog box with the specified destination variable name, button text and style.
+		/// The supported button styles can be accessed through const strings on the Style.Button class.
+		/// </summary>
+		/// <param name="destVar">The name of the destination variable.</param>
+		/// <param name="displayText">The button text.</param>
+		/// <param name="style">The button style (see Style.Button for supported styled).</param>
+		/// <returns>This <see cref="UIBuilder"/> instance.</returns>
+		/// <remarks>
+		/// <para>When the button is pressed, the destination variable is filled with its own name.</para>
+		/// <para>This is a convenience method that will create a new instance of <see cref="UIBlockDefinition"/> of type Button and sets <see cref="RequireResponse"/> to <c>true</c>.</para>
+		/// </remarks>
+		/// <example>
+		/// <code>
+		/// UIBuilder uibDialogBox1 = new UIBuilder();
```

Comment:
The example doesn't compile. Here we use "uibDialogBox1", but in the next lines, we use uiBuilder. We should make it consistent. Note: this is the case as well for the other AppendButton example